api/proto/info/info.proto

```diff
@@ -243,6 +243,27 @@ message ZInfoAppInstance {
   string domainName = 3;
 }
 
+enum ZSimcardState {
+  SIM_INVALID     = 0;
```

Comment:
We normally reserve zero for "none". Here doesn't INVALID mean that the sim is invalid? or is it that the state is invalid?

What about the case where there is no modem? What do you set then?

Reply:
Invalid is same as NONE. Will change it to None.
If there is no Modem, there would not be any Sim Cards.. The array sims  would be of zero length

Reply:
Yetus forcing me to use INVALID. Changing it back to Invalid.:

info/info.proto:247:3:Zero value enum field "Z_SIMCARD_STATE_NONE" is expected to have the name "Z_SIMCARD_STATE_INVALID".